superduperdb/base/config.py

```diff
@@ -92,6 +92,13 @@ class Logging(JSONable):
     kwargs: dict = Factory(dict)
 
 
+class VectorSearchServer(JSONable):
+    host: str = '0.0.0.0'
+    port: int = 8000
+    protocol: str = 'http'
+    name: str = 'vector_search'
+
+
 class Server(JSONable):
```

Comment:
This will soon be deprecated, since server will go to an alternative repository.